client.go

```diff
@@ -2,6 +2,7 @@ package sdk
 
 import (
 	"fmt"
+	"github.com/bianjieai/irita-sdk-go/modules/oracle"
```

Comment:
make format

Reply:
The code have been formatted and passed the test.